dev/tools/gen_defaults/lib/tabs_template.dart

```diff
@@ -21,7 +21,7 @@ class _${blockName}DefaultsM3 extends TabBarTheme {
   late final TextTheme _textTheme = Theme.of(context).textTheme;
 
   @override
-  Color? get dividerColor => ${componentColor("md.comp.primary-navigation-tab.divider")};
+  Color? get dividerColor => _colors.surfaceVariant;
```

Comment:
It's possible the guidelines are wrong, could you please revert this for the time being and file a bug at go/guidelinebug?

Reply:
I see. Just reverted the token and the template. Also filed a bug here: b/273575840. Thanks!